website/content/en/docs/cli/operator-sdk.md

```diff
@@ -60,7 +60,10 @@ to obtain further info about available commands.
 * [operator-sdk edit](../operator-sdk_edit)	 - This command will edit the project configuration
 * [operator-sdk generate](../operator-sdk_generate)	 - Invokes a specific generator
 * [operator-sdk init](../operator-sdk_init)	 - Initialize a new project
-* [operator-sdk olm](../operator-sdk_olm)	 - Manage the Operator Lifecycle Manager installation in your cluster
+* [operator-sdk olm](../operator-sdk_olm)	 - Manage the Operator Lifecycle Manager installation in your cluster.
+
+Operator SDK officially supports the following OLM versions: unknown.
```

Comment:
Why does this say unknown?

Reply:
Ops, I missed this. Will look into it

Reply:
Because the list is built at compile time, not dynamically. The list would have to be encoded directly as a string.

Reply:
Wouldn't that be an added task of updating the list manually when we update bindata while making a release ? 
Do you think it would be better if we just document this somewhere in olm integration docs, instead of adding in help text ?

Reply:
Backtracking on my previous thought, documenting this in https://master.sdk.operatorframework.io/docs/overview/ is ok. Can you add a step to the minor version release doc about making sure `OLM_VERSIONS` and that doc list align?